pkg/network/render.go

```diff
@@ -385,6 +388,59 @@ func isNetworkChangeSafe(prev, next *operv1.NetworkSpec, infraRes *bootstrap.Inf
 	return nil
 }
 
+func isClusterNetworkChangeSafe(prev, next *operv1.NetworkSpec) bool {
```

Comment:
can we stick to `isXXXChangeSafe(...) error` signature? We can then just return a appropriate error instead of just logging.